plugin/src/main/scala/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -8,10 +8,15 @@ import nsc.plugins.{Plugin, PluginComponent}
 import scala.reflect.internal.util.NoPosition
 import scala.collection.mutable
 
-private[plugin] case class ChiselPluginArguments(val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty) {
+private[plugin] case class ChiselPluginArguments(
+  val skipFiles:                mutable.HashSet[String] = mutable.HashSet.empty,
+  var buildElementsAccessor:    Boolean = true,
+  var pluginDebugBundlePattern: String = "" // For example try .*Bpip.*"
+) {
   def useBundlePluginOpt = "useBundlePlugin"
   def useBundlePluginFullOpt = s"-P:${ChiselPlugin.name}:$useBundlePluginOpt"
-
+  def buildElementAccessorOpt = "buildElementAccessor"
+  def buildElementAccessorFullOpt = s"-P:chiselplugin:$buildElementAccessorOpt"
```

Comment:
```suggestion
  def genBundleElementsOpt = "genBundleElements"
  def genBundleElementsOpt = s"-P:chiselplugin:$genBundleElementsOpt"
```
I would reframe this more toward the user API than the internals (what does "Accessor" mean to the user?) I'm basing this off the 3.4.3 option `-P:chiselplugin:useBundlePlugin`, but am open to other ideas.

Reply:
Works for me, will do this